library/std/src/error.rs

```diff
@@ -637,6 +637,14 @@ impl Error for core::ffi::FromBytesWithNulError {
     }
 }
 
+#[unstable(feature = "todo", issue = "none")]
```

Comment:
This should be marked stable; trait impls like this can't be unstable.

```suggestion
#[stable(feature = "io_error_from_frombyteswithnulerror", since = "1.62.0")]
```